src/mono/CMakeLists.txt

```diff
@@ -202,16 +211,37 @@ elseif(CMAKE_SYSTEM_NAME STREQUAL "iOS")
   set(DISABLE_CRASH_REPORTING 1)
   set(ENABLE_MONOTOUCH 1)
   add_definitions(-DMONOTOUCH=1)
-  add_definitions(-DHOST_IOS=1)
   set(HAVE_SYSTEM 0)
   set(MONO_DL_NEED_USCORE 1)
+  if(CMAKE_SYSTEM_NAME STREQUAL "tvOS")
+	set(HOST_TVOS 1)
```

Comment:
Tab again. Maybe specify the cmake files in the editorconfig as well, as a followup to Aleksey's PR?